examples/storybook/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "storybook",
+  "version": "0.0.1",
+  "scripts": {
+    "start": "start-storybook -p 9001 -c .storybook",
+    "build": "build-storybook -c .storybook -o dist/"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/common": "7.0.0",
+    "@angular/compiler": "7.0.2",
+    "@angular/core": "7.0.0",
+    "@angular/forms": "7.0.2",
+    "@angular/platform-browser": "7.0.2",
+    "@angular/platform-browser-dynamic": "7.0.2",
+    "angular-instantsearch": "2.1.0",
+    "core-js": "2.5.4",
+    "instantsearch.js": "2.10.4",
+    "rxjs": "6.3.3",
+    "zone.js": "0.8.26"
+  },
+  "devDependencies": {
```

Comment:
Since a storybook can be seen as an application, should we pin the dev dependencies?

Reply:
devDependencies don't have the potential to break something in production. Therefore there is no risk is not pinning them.

Every example right now has pinned dependencies and range devDependencies. This is one of the first PRs I made on this project, following the recommendation in this read https://renovatebot.com/docs/dependency-pinning/#reducing-the-noise-of-dependency-updates

Reply:
An example is not `require`able, thus it should in theory follow the rule of only pinned dependencies.

I agree though that unpinning them will reduce PR noise.

Reply:
At least they can share all the same pattern for the range.